pkg/server/server.go

```diff
@@ -85,9 +86,10 @@ func New(ctx context.Context, log *zap.Logger, options Options) (*Server, error)
 
 	s.ctx, s.cancel = context.WithCancel(ctx)
 
+	promReg := prometheus.NewRegistry()
```

Comment:
Had to do a bit of work with metrics to consolidate into the same prometheus registry or else we're losing the api metrics.

Reply:
We'll have to update the DD references to use the `xmtpd.` prefix on metrics instead of `waku.` prefix after merging this